### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by helenabarmer** [shashigharti/federated-learning-on-raspberry-pi#21]
/Helena Barmer

**Comment by jess-s** [comment]
What does this mean?

**Comment by helenabarmer** [comment]
@jess-s Updating the scope and discreption mentioned in the Project Showcase Challenge outlines 👍

**Comment by Joycechidi** [comment]
@Joyce

**Comment by helenabarmer** [comment]
@Joycechidi I have assigned you 👍

**Comment by jess-s** [comment]
It seems like this has been updated?

**Comment by Joycechidi** [comment]
@Jess, do you mean that it has been updated by someone else apart f om the update I made today?

**Comment by jess-s** [comment]
@Joycechidi No, I was commenting that it appears "done" unless you want to add more

**Comment by Joycechidi** [comment]
Okay. I get what you mean
